src/components/CustomerQuota/CheckoutUnsuccessful/CheckoutUnsuccessfulCard.tsx

```diff
@@ -0,0 +1,179 @@
+  useEffect(() => {
+    if (error) {
+      showErrorAlert(error);
+    }
+  }, [error, showErrorAlert]);
+
+  const latestTransactionTime: Date | undefined =
+    (quotaResponse && getLatestTransactionTime(cart)) ?? undefined;
+
+  const translationProps = useTranslate();
+  const { c13nt, i18nt } = translationProps;
+  const transactionsByCategoryMap = groupTransactionsByCategory(
+    sortedTransactions,
+    allProducts || [],
+    latestTransactionTime,
+    translationProps
+  );
+
+  const transactionsByCategoryList = sortTransactions(
+    transactionsByCategoryMap
+  );
+  return (
+    <View>
+      <CustomerCard ids={ids} headerBackgroundColor={color("red", 60)}>
+        <View style={sharedStyles.failureResultWrapper}>
+          <View style={sharedStyles.resultWrapper}>
+            <FontAwesome
+              name="thumbs-down"
+              color={color("red", 60)}
+              style={sharedStyles.icon}
+            />
+            <View style={sharedStyles.statusTitleWrapper}>
+              <AppText
+                style={sharedStyles.statusTitle}
+                accessibilityLabel="checkout-unsuccessful-title"
+                testID="checkout-unsuccessful-title"
+                accessible={true}
+              >
+                {`${c13nt(
+                  "checkoutUnsuccessfulTitle",
+                  undefined,
+                  i18nt("checkoutUnsuccessfulScreen", "unsuccessful")
+                )}`}
+              </AppText>
+            </View>
+            <View>
+              <AppText style={{ marginBottom: size(1) }}>
+                {`${c13nt(
+                  "checkoutUnsuccessfulDescription",
+                  undefined,
+                  i18nt("checkoutUnsuccessfulScreen", "logAppeal")
+                )}.`}
+              </AppText>
+            </View>
+            <View>
+              <View style={styles.checkoutItemsList}>
+                {loading ? (
+                  <ActivityIndicator
+                    style={{ alignSelf: "flex-start" }}
+                    size="large"
+                    color={color("grey", 40)}
+                  />
+                ) : (
+                  <View>
+                    <AppText>
+                      {`${i18nt(
+                        "checkoutUnsuccessfulScreen",
+                        "unsuccessfulRedeemAttempt"
+                      )}:`}
+                    </AppText>
+                    {transactionsByCategoryList.map(
+                      (
+                        transactionsByCategory: TransactionsGroup,
+                        index: number
+                      ) => (
+                        <TransactionsGroup
+                          key={index}
+                          maxTransactionsToDisplay={
+                            isShowFullList
+                              ? BIG_NUMBER
+                              : MAX_TRANSACTIONS_TO_DISPLAY
+                          }
```

Comment:
I noticed this approach is different from `CheckoutSuccessCard`, can we apply this for `CheckoutSuccessCard` so there's no need to do the slice there? If so, please make a new PR to remove that